mne_qt_browser/_pg_figure.py

```diff
@@ -3504,6 +3507,9 @@ def change_duration(self, checked=False, *, step):
         if xmin < 0:
             xmin = 0
 
+        if self.mne.filter_coefs is not None:
+            self._rerun_precompute()
```

Comment:
Same -- this seems like the wrong place to need to rerun precomputation. Shouldn't this just be run when how the signal is processed (e.g., projectors, whitening, etc.) is changed rather than being done inside a duration change event?